wheatley/main.py

```diff
@@ -28,14 +28,9 @@
 
 def create_row_generator(args: argparse.Namespace) -> RowGenerator:
     """ Generates a row generator according to the given CLI arguments. """
-    if "complib_share_link" in args and args.complib_share_link is not None:
+    if "comp" in args and args.comp is not None:
         try:
-            row_gen = ComplibCompositionGenerator.from_url(args.complib_share_link)
-        except (PrivateCompError, InvalidCompError) as e:
-            sys.exit(f"Bad value for '--comp': {e}")
-    elif "comp" in args and args.comp is not None:
-        try:
-            row_gen = ComplibCompositionGenerator(args.comp)
+            row_gen = ComplibCompositionGenerator.from_arg(args.comp)
```

Comment:
Sorry, looks like pylint correctly doesn't like `row_gen` initally be declared with the more specific type.
Replace `row_gen = ` with `return`?

Reply:
Hmm hopefully this should fix it...

Reply:
Ayyy fixed :tada:!